src/commands/init.ts

```diff
@@ -19,13 +22,22 @@ export class InitCommand implements Command {
 
   description = 'Initializes a Polymer project';
 
-  args = [];
+  args: ArgDescriptor[] = [
+    {
+      name: 'name',
+      description: 'The template name',
+      type: String,
+      defaultOption: true,
+    },
+  ];
 
   run(options): Promise<any> {
     return new Promise((resolve, reject) => {
+      let templateName = options['name'] || 'polymer-init';
       let env = new YeomanEnvironment();
       env.registerStub(polymerGenerator, 'polymer-init:app');
-      env.run('polymer-init', {}, () => resolve());
+      env.registerStub(PolykartGenerator, 'polykart:app');
```

Comment:
You mentioned that the line above just broke for you, rebase and use the `env.register()` method instead.